jdi-light/src/main/java/com/epam/jdi/light/elements/common/Keyboard.java

```diff
@@ -0,0 +1,78 @@
+package com.epam.jdi.light.elements.common;
+
+import com.epam.jdi.tools.LinqUtils;
+import com.epam.jdi.tools.ReflectionUtils;
+import com.epam.jdi.tools.Timer;
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.awt.*;
+import java.awt.datatransfer.Clipboard;
+import java.awt.datatransfer.StringSelection;
+import java.awt.event.InputEvent;
+import java.awt.event.KeyEvent;
+import java.lang.reflect.Field;
+import java.util.List;
+
+import static com.epam.jdi.light.common.Exceptions.exception;
+import static com.epam.jdi.tools.LinqUtils.first;
+import static java.awt.event.KeyEvent.*;
+import static java.lang.reflect.Modifier.isStatic;
+
+public class Keyboard {
+    static Robot robot;
+    static Robot getRobot() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4958360)